openshift-sdn.spec

```diff
@@ -0,0 +1,141 @@
+#debuginfo not supported with Go
+%global debug_package %{nil}
+%global gopath      %{_datadir}/gocode
+%global osdn_gopath _output/local/go
+%global gopkg_base  github.com/openshift
+%global import_path %{gopkg_base}/openshift-sdn
+%global commit      1d03a6c75a44a638b9f2590b1475a6c980ce829d
```

Comment:
no such a commit "1d03a6c75a44a638b9f2590b1475a6c980ce829d" in openshift-sdn git log.
It means conflict with tar download with link written in Source0